src/modules/auth/email/EmailScreen.js

```diff
@@ -43,7 +43,7 @@ function ManageEmailCard(props) {
 	let redirect_uri = params.get("redirect_uri");
 
 	let history = useHistory();
-	let SeaCatAuthAPI = props.app.axiosCreate('seacat-auth');
+	let SeaCatAccountAPI = props.app.axiosCreate('seacat-auth/account');
```

Comment:
@byewokko I dont think this is a good way to do this. Here should be just a service name. So only `seacat-auth` here should be correct.

This is the correct way:

```
response = await SeaCatAccountAPI.put("/account/credentials", ...
```

Reply:
the idea was to have a separate axios instance for login and auth stuff (`seacat-auth/public`) and another one for the account management (`seacat-auth/account`). they now even have different authorization and ports on backend. i wanted to make this separation clear in the code:
```js
let SeaCatAccountAPI = props.app.axiosCreate('seacat-auth/account');
let SeaCatPublicAPI = props.app.axiosCreate('seacat-auth/public');
```
but maybe it is not necessary. what does @ateska think?

Reply:
As discussed: Franta is right. There is no need for separation, also the proposed changes would make the code harder to search and correlate with the nginx config. So the correct approach is this:
```js
let SeaCatAuthAPI = props.app.axiosCreate('seacat-auth');
...
response = await SeaCatAuthAPI.put("/account/credentials", ...)
```